src/robot/running/librarykeywordrunner.py

```diff
@@ -119,9 +119,12 @@ def _dry_run(self, context, args):
             self._handler.resolve_arguments(args)
 
     def _executed_in_dry_run(self, handler):
+        keywords_to_execute = ('BuiltIn.Import Library',
+                                          'BuiltIn.Set Library Search Order',
+                                          'BuiltIn.Set Tags',
+                                          'BuiltIn.Remove Tags')
```

Comment:
Indentation looks strange. Names should be aligned like
```python
keywords_to_execute = ('BuiltIn.Import Library',
                       'BuiltIn.Set Library Search Order',
                       'BuiltIn.Set Tags',
                       'BuiltIn.Remove Tags')
```

Did I already mention we are picky about whitespace? :smile: